lib/pretty_text.rb

```diff
@@ -442,6 +443,21 @@ def self.extract_links(html)
     links
   end
 
+  def self.add_video_placeholder_image(doc)
+    doc
+      .css(".video-placeholder-container")
+      .each do |video|
+        video_src = video["data-video-src"]
+        video_sha1 = File.basename(video_src, File.extname(video_src))
+        thumbnail = Upload.where("original_filename LIKE ?", "#{video_sha1}.%").last
+        if thumbnail
+          video[
+            "data-thumbnail-src"
+          ] = "/uploads/default/original/1X/#{thumbnail.sha1}.#{thumbnail.extension}"
```

Comment:
Is this multisite and/or subfolder safe? `/uploads/default` might not work on either.

Reply:
Oh ya, I forgot I hard coded these. Let me see what I can improve.

Reply:
I'm not totally sure the correct way to go about doing this, but I switched to using the upload.url and using the UrlHelper to generate the url like we are doing in other places.